js/sign/README.md

```diff
@@ -104,14 +104,18 @@ This package also includes 2 CLI tools
 There are the following command-line flags available:
 
 - (required) `--privateKey <filePath>` (`-k <filePath>`)  
-  which takes the path to ed25519 private key.
+  which takes the path to ed25519 private key. This can be specified multiple times.
 - (required) `--input <filePath>` (`-i <filePath>`)  
   which takes the path to the web bundle to be signed.
 - (optional) `--output <filePath>` (`-o <filePath>`)  
   which takes the path to the wanted signed web bundle output. Default:
   `signed.swbn`.
+- (optional) `--version <version>`  
+  which can be either `v1` or `v2`, defaulting to `v1`. Sets the integrity block format.
+- (required if `v2`) `--web-bundle-id <web-bundle-id>`  
```

Comment:
not exactly -- this is required only if there's more than one key specified.

Reply:
Corrected